packages/app/app.py

```diff
@@ -1,11 +1,46 @@
 from flask import Flask, render_template, request, redirect, url_for
 import sqlite3 as sql
 from sqlite3 import Error
+import requests
+import json
+import os
+from dotenv import load_dotenv
+
+load_dotenv()
 
 # Configure application
 app = Flask(__name__)
 
-# Connect to db function
+def yelp_lookup(city, state):
```

Comment:
I thought we talked about setting up a scheduled task to perform the database seeding, is that no longer your plan?

Reply:
Yes. The plan is to use the scheduler as we discussed. I renamed this branch after ticket IAM-705 was created last Monday. Prior to the branch being named as IAM-705, I had been testing out Yelp API calls/Different scenarios b/c I was unsure about how to proceed. I am following the plan we discussed/outlined in IAM-705 and have updated the final project sequence diagram accordingly.